Package.swift

```diff
@@ -18,13 +18,18 @@ let package = Package(
     .package(
       url: "https://github.com/layoutBox/FlexLayout",
       .upToNextMajor(from: "1.3.0")
+    ),
+    .package(
+      url: "https://github.com/layoutBox/FlexLayout",
```

Comment:
https://github.com/layoutBox/PinLayout

It needs to be modified to this.
After changing it, you can update the package.resolve by reloading the swift package.